src/com/owncloud/android/files/FileMenuFilter.java

```diff
@@ -134,30 +137,34 @@ private void hideAll(Menu menu) {
      * @param toHide            List to save the options that must be shown in the menu.
      */
     private void filter(List<Integer> toShow, List <Integer> toHide) {
+
         boolean synchronizing = anyFileSynchronizing();
 
+        boolean videoPreviewing = anyFileVideoPreviewing();
```

Comment:
Then we do not allow anymore to rename or copy the file in the preview mode. But we allowed it before. Why not now?